packages/calypso-stripe/package.json

```diff
@@ -43,12 +43,5 @@
 		"react-dom": "^16.12.0",
 		"react-stripe-elements": "^4.0.2"
 	},
-	"devDependencies": {
```

Comment:
This package is private, but if it were not, would we want to have these `devDependencies` included?

Reply:
I removed the dependencies not because the package is private, but because it is not using them. It has no tests whatsoever, so there is no point in importing `@testing-library/*`. Also it is not calling `webpack` in any way, so no need to import `webpack`, `cache-loader` or `@automattic/calypso-polyfills`